content/pypi-attestations-ga.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "PyPI's Sigstore-powered attestations are now generally available"
+date = "2024-11-14"
+tags = ["sigstore", "security", "pypi", "python"]
+draft = false
+author = "William Woodruff (Trail of Bits), Dustin Ingram (Google), Hayden Blauzvern (Google)"
+type = "post"
++++
+
+*Check out the [PyPI blog] and [Trail of Bits blog] for more user-facing and
+technical details, respectively!*
+
+Over the past year, the Google Open Source Security Team
+and [Trail of Bits] have worked together to implement [PEP 740],
+a Python packaging standard that allows users to upload Sigstore-based
+attestations to the [Python Package Index].
+
+Today we're pleased to announce that attestation support on PyPI is
+**generally available**, meaning that package maintainers
+can submit attestations for both PyPI and downstream users to verify.
+
+An important piece of the story for attestations on PyPI is
+**default enablement**: if a project uses [Trusted Publishing],
+simply upgrading to [`pypa/gh-action-pypi-publish`] [v1.11.0] or newer will result
+in attestation generation by default, with no changes required.
```

Comment:
This might be a bit too much detail for the Sigstore blog? Readers might not know what `pypa/gh-action-pypi-publish` is. Maybe we can just say "if a project uses Trusted Publishing and the [canonical GitHub Action](link) to upload their distribution" or something like that.

Reply:
Done, PTAL 🙂